### ISSUE TITLE
Staging Review finding: Ambiguous instructions for providing feedback

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#84221]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV-Medications
**Product:** MHV Medications
**Feature:** Medications

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/medications/about](https://staging.va.gov/my-health/medications/about) in the "What to know" section, the text says:
> We need your feedback to keep making this tool better for you and all Veterans. Send us your feedback and questions using the feedback button on this page.

There are two feedback buttons on each page -- the feedback button at the bottom of every page on VA.gov, and the feedback button rotated 90 degrees along the right-hand edge of the page. These buttons are visually different but it's not clear if they do different things, or which button Veterans should use to provide their thoughts.

No idea where the rotated feedback button came from, but this is the first time I'm seeing it?

<details><summary>Link, screenshot or steps to recreate</summary>

![Two feedback buttons visible on the page. One is oriented horizontally and is styled to match other VA.gov buttons. The other is rotated by 90 degrees and diverges from other buttons in color, typography, and shape.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/58241-Brian-1.png?raw=true)

</details>

## Recommended action

If this new feedback button is an MHV thing, please refer this to the appropriate team. If it's not an MHV thing, then this is probably out of scope for your team and we'll try to track this down.

If these buttons serve identical functions, is it necessary to have both on the page?

If these buttons do something different, the button text should be made distinct, eg. "Feedback" and "MHV Feedback" or something similar.

I don't know if it would be considered within the scope of this review, but I'll also note a few concerns I have with the feedback button along the right edge of the page:
- Styling that's inconsistent with other buttons on VA.gov may make it more difficult for voice command users to identify as a button, which may prevent them from using it.
- The keyboard focus style is inconsistent with other elements on VA.gov and may not meet current WCAG focus requirements.
- Sticky elements can present issues for low vision users (eg. obscuring other elements on the page at high browser zoom, not easy to find for screen magnification users).
- Rotating text by 90 degrees may increase cognitive load for some users.

If there's space to reexamine this element --- both the button text and the overall design and placement of it --- I would encourage you to do so.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen magnification
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by IdeaTrail** [comment]
Completed with Jira item: https://jira.devops.va.gov/browse/MHV-58724
